packages/bierzo-wallet/src/routes/payment/index.dom.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { TokenTicker } from '@iov/bcp';
+import TestUtils from 'react-dom/test-utils';
+import { DeepPartial, Store } from 'redux';
+
+import { aNewStore } from '../../store';
+import { BalanceState } from '../../store/balances';
+import { RootState } from '../../store/reducers';
+import { click, expectRoute } from '../../utils/test/dom';
+import { PAYMENT_ROUTE } from '../paths';
+import { getCancelButton, getSelectedCurrency } from './test/operatePayment';
+import { travelToPayment } from './test/travelToPayment';
+
+const balancesAmount: DeepPartial<BalanceState> = {
+  BASH: {
+    quantity: '82500',
+    fractionalDigits: 4,
+    tokenTicker: 'BASH' as TokenTicker,
+  },
+  CASH: {
+    quantity: '1226775',
+    fractionalDigits: 5,
+    tokenTicker: 'CASH' as TokenTicker,
+  },
+};
+
+describe('The /payment route', () => {
+  let store: Store<RootState>;
+  let paymentDom: React.Component;
+  beforeEach(
+    async (): Promise<void> => {
+      store = aNewStore({
+        extension: {
+          connected: true,
+          installed: true,
+        },
+        balances: balancesAmount,
+      });
+      paymentDom = await travelToPayment(store);
+    },
+  );
+
+  it('should redirect to the /balance route when cancel clicked', async () => {
```

Comment:
@albertandrejev Write in the new ticket you are about to start to update this.

Reply:
Hmmm.. I think you are right.